apps/freeablo/faworld/missile/missilecreation.cpp

```diff
@@ -42,4 +44,28 @@ namespace FAWorld::Missile
     {
         missile.mGraphics.push_back(std::make_unique<MissileGraphic>("", missile.getGraphicsPath(0), std::nullopt, Position(missile.mSrcPoint), level));
     }
+
+    void Missile::Creation::townPortal(Missile& missile, Misc::Point, GameLevel* level)
+    {
+        // Add portal near player
+        auto noMissilesAtPoint = [&level](const Misc::Point& p) {
+          return std::none_of(
+              level->mMissileGraphics.begin(), level->mMissileGraphics.end(),
+              [&p](const MissileGraphic* g) { return p == g->mCurPos.current();});
+        };
+        auto point = level->getFreeSpotNear(missile.mSrcPoint, INT32_MAX, noMissilesAtPoint);
```

Comment:
Not a huge deal, but I think it's better to use std::numeric_limits<int32_t>::max() than INT32_MAX